specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-03-01-preview/subscriptions.json

```diff
@@ -172,6 +172,12 @@
             "MS-AZR-0017P",
             "MS-AZR-0148P"
           ]
+        },
+        "additionalParameters":{  
```

Comment:
You have an actual "additionalParameters" node in your JSON? Could you update an example with this node, I'm always more confident when I see an example if "additionalProperties" is used (regular source of trouble :))

Reply:
Looks like I messed it up too. :) Added an example and made sure it's passing validation.